src/app/shared/view-issue/new-team-response/new-team-response.component.ts

```diff
@@ -100,7 +100,7 @@ export class NewTeamResponseComponent implements OnInit {
       finalize(() => this.isFormPending = false)
     ).subscribe((resultArr: [Issue, IssueComment]) => {
         const [updatedIssue, updatedComment] = resultArr;
-      updatedIssue.teamResponse = this.description.value;
+      updatedIssue.teamResponse = Issue.updateDescription(this.description.value);
```

Comment:
@JunWei96, I believe we need to check for the empty description here as well, in order to ensure the local store is updated correctly?

Reply:
Actually, the updated Issue returned by the issue service is already in the correct format. There is no need for line 103 & 104. Can help to delete them? Thanks!

Reply:
> Can help to delete them?

Done

Reply:
I can't reproduce the bug you mentioned. The empty string of team response is probably treated as a "False" value. 

![Screen Recording 2020-02-18 at 1 45 06 PM](https://user-images.githubusercontent.com/22557857/74707704-81b45b80-5255-11ea-9f16-cd649f7dd3f4.gif)

Reply:
> With this change, I can't reproduce the bug you mentioned.

Thanks for testing it out. This is somewhat strange, as I'm still seeing this bug.
Let me try investigating it and I'll update you on what I find?